views/components/applicationSummary.njk

```diff
@@ -0,0 +1,19 @@
+{% macro readOnlySummary(summary) %}
+    {% for step in summary %}
+        <h4 class="data__step">{{ step.name }}</h4>
```

Comment:
These `data__` classes look like they are being used out of context. I can't see a parent `data` class anywhere? Do these need to be different classes?